nss-cohort-36/bangazon-api-imagination-station-api
# Rc fixtures

# Description

I've created the fixtures for when someone clones down the repo. It creates a user that has orders, products, etc. It creates multiple product types. When the repo is cloned down, the user will have to register a new account, but can see the products of the test user.

Fixes #91 

## Type of change

Adding fixtures, re-creating database.

# Testing Instructions

MAKE SURE TO COPY YOUR db.sqlite3 FILE TO ANOTHER FOLDER IN CASE THIS BLOWS UP!!!

Add and Commit your current branch.
`git checkout master`
`git fetch --all`
`git checkout rc-fixtures`

Copy db.sqlite3 file and paste it in a folder on your computer, like documents.
Delete the db.sqlite3 file in your bangazon folder.
`python manage.py makemigrations` or `python manage.py makemigrations bangazonapi`
`python manage.py migrate` or `python manage.py migrate bangazonapi`
`python manage.py loaddata */fixtures/*.json`
`python manage.py test`

All of your tests should pass.
You should now have a working database that has an existing user. You will need to register a new user. Try out the app and make sure everything seems to work!

# Checklist:
